flow-server/src/main/java/com/vaadin/flow/dom/Style.java

```diff
@@ -183,6 +183,18 @@ default Style setBorderBottom(String value) {
         return set(STYLE_BORDER_BOTTOM, value);
     }
 
+    /**
+     * Sets the <code>border-radius</code> property.
+     *
+     * @param value
+     *            the style property value (if <code>null</code>, the property
+     *            will be removed)
+     * @return this style instance
+     */
+    default Style setBorderRadius(String value) {
+        return set(STYLE_BORDER_RADIUS,value);
```

Comment:
Fixing formatting issue

```suggestion
        return set(STYLE_BORDER_RADIUS, value);
```